modules/custom/az_migration/az_migration.info.yml

```diff
@@ -0,0 +1,11 @@
+name: AZ Quickstart Migration
+description: 'Provides migrations for the site.'
+core_version_requirement: ^8.8 || ^9
+type: module
+package: The University of Arizona
+dependencies:
+ - drupal:migrate
+ - drupal:migrate_plus
+ - drupal:migrate_tools
+ - drupal:migrate_drupal
+ - drupal:media_migration
```

Comment:
Should we declare any of the modules that provide our destination entity bundles/fields as dependencies here? (e.g. `az_person`, `az_cas`, etc)

Reply:
I think, .info file should be good place to declare it, I will make that change and push it.

Reply:
I agree with that, I think declaring the `az_person` etc. modules here would be good.